packages/data/src/namespace-store/index.js

```diff
@@ -27,29 +36,46 @@ export default function createNamespace( key, options, registry ) {
 	const reducer = options.reducer;
 	const store = createReduxStore( key, options, registry );
 
-	let selectors, actions, resolvers;
-	if ( options.actions ) {
-		actions = mapActions( options.actions, store );
-	}
-	if ( options.selectors ) {
-		selectors = mapSelectors( options.selectors, store, registry );
-	}
+	let resolvers;
+	const actions = mapActions( {
+		...metadataActions,
```

Comment:
An initial thought occurred to me that there's risk for conflict between these metadata actions and a store's own. On further consideration, I don't see it likely to occur, or at least it should be known that these "baseline" actions/selectors exist for every store. Which leads me to wonder: How do we document this?

Reply:
yes, I assume these can't be auto-documented. I'll see if I can add a note in the data module README (where we document `registerStore`)

Reply:
> yes, I assume these can't be auto-documented.

I'm not so overly concerned about the selectors themselves (though it would be nice), more to just general awareness that this is a behavior to expect, the fact that resolution behavior is included in every store by default.

Reply:
There's potential side-effects here of consuming code being able to "override" default metadata selectors and actions.  I almost wonder if there should be some enforcement here: i.e. checking the incoming actions/selectors and ensuring there are no collisions with the corresponding metadata functions?

Reply:
If we're not confident exposing these selectors and actions yet, we can easily add `__internal` or `__experimental` for now, what do you think?

Reply:
I'm less concerned with exposing them (I think they _should_ be exposed?) and more concerned with potential conflicts.  

I do think it'd be better (at least initially) to do this:

```
// for actions
const actions = mapActions( {
    ...options.actions,
    ...metadataActions,
} );
// and something similar with selectors reversing the order
```

That way the metadataActions and metadataSelectors take precedence.  Of course you may _intentionally_ have them this way so they can be overridden by the incoming store config but if not, I think this is safer.